logfire/_internal/cli.py

```diff
@@ -246,7 +272,7 @@ def parse_auth(args: argparse.Namespace) -> None:
 
 def parse_list_projects(args: argparse.Namespace) -> None:
     """List user projects."""
-    logfire_url = args.logfire_url
+    logfire_url = _get_logfire_url(args.logfire_url, args.region)
```

Comment:
It'd be nice to not repeat this for every command.

Reply:
I initially wanted to set `namespace.logfire_url = _get_logfire_url(args.logfire_url, args.region)` once parsed (and before dispatching to the subcommand callback) but thought it would be confusing as this "overrides" the explicitly provided `logfire_url`. Anyway, I went with the change.